__tests__/app.test.js

```diff
@@ -71,4 +71,69 @@ describe('app', () => {
         });
     });
   });
+  describe('PATCH -  /api/articles/:article_id', () => {
+    //404 and 500 already covered
+    test('Status: 200 - should respond with updated article', () => {
+      const voteUpdate = { inc_votes: 10 };
+      return request(app)
+        .patch('/api/articles/1')
+        .send(voteUpdate)
+        .expect(200)
+        .then(({ body: article }) => {
+          expect.objectContaining({
+            article: {
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(Number),
+              votes: expect.any(Number),
+            },
+          });
+          expect(article.article.votes).toBe(110);
+        });
+    });
+    test('Status: 200 - should respond with updated article if number passed is negative', () => {
+      const voteUpdate = { inc_votes: -10 };
+      return request(app)
+        .patch('/api/articles/1')
+        .send(voteUpdate)
+        .expect(200)
+        .then(({ body: article }) => {
+          expect.objectContaining({
+            article: {
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(Number),
+              votes: expect.any(Number),
+            },
+          });
+          expect(article.article.votes).toBe(90);
```

Comment:
you're checking your response, can you check in the db is actually being updated somehow?